### PR TITLE
update gitrepositories.md (fixes #588)

### LINKED ISSUE #588: Layout Improvement for Git Repositories Page (Step 5 - Keeping Fork Updated)
### Problem
Position of the **Summary of Steps** section could possibly be adjusted; summary at the bottom of the page tends to be **overlooked**

 ### Steps to reproduce the problem
Visit [https://treehouses.io/#!./pages/vi/gitrepositories.md](https://treehouses.io/#!./pages/vi/gitrepositories.md)

 ### Screenshots
image

 ### Proposed solution
**Summary of Steps** should be placed at the **top** instead of the **bottom** of the page

### PR DESCRIPTION
**Pull request by sydneyaldo**
Fixes #588 

### Description
Changed the position of the Summary of Steps 

### Raw.Githack preview link
http://raw.githack.com/sydneyaldo/sydneyaldo.github.io/issue-%23588/#!pages/vi/gitrepositories.md

### CODE REVIEW

File: pages/vi/gitrepositories.md [hunk 1]
```diff
@@ -12,8 +12,59 @@ On GitHub, our software code is organized in repositories, each of which represe
 
 You previously forked a repository from treehouses to work on, which means that that a version of the treehouses repository was available on your own GitHub account. You were then able to modify your fork, then send an update to the upstream repository in the form of a pull request. 
 
+## Summary of Steps
+Generally, this is what the structure of your workflow will look like: 
+
+![Image](images/upstream_diagram.png)
```
Comment (dogi) [inline pages/vi/gitrepositories.md:18]: @sydneyaldo think this file is missing

Comment (sydneyaldo) [reply pages/vi/gitrepositories.md:18]: The picture file is missing?

Comment (sydneyaldo) [reply pages/vi/gitrepositories.md:18]: All I did was change the position, it shouldn't be missing

Comment (dogi) [reply pages/vi/gitrepositories.md:18]: oops sorry you are right ;)

**Review by dogi — APPROVED** [review]

**Pull request merged**